apps/storybook/src/TransferList.stories.tsx

```diff
@@ -0,0 +1,288 @@
+                actionable
+                active={item.active}
+                onActiveChange={(isActive: boolean) => {
+                  setAppliedData((oldData) => {
+                    const newData = [...oldData];
+                    const newObject = { ...newData[index] };
+                    newObject.active = isActive;
+                    newData[index] = newObject;
+                    return newData;
+                  });
+                }}
+              >
+                {item.name}
+              </TransferList.Item>
+            );
+          })}
+        </TransferList.Listbox>
+      </TransferList.ListboxWrapper>
+    </TransferList>
+  );
+};
+Basic.args = {};
+
+export const WithLabel: Story<TransferListProps> = (args) => {
+  type TransferItemDataType = {
+    name: string;
+    active: boolean;
+  };
+
+  const [optionData, setOptionData] = React.useState([
+    { name: 'Option 1', active: false },
+    { name: 'Option 2', active: false },
+    { name: 'Option 3', active: false },
+    { name: 'Option 4', active: false },
+    { name: 'Option 5', active: false },
+    { name: 'Option 6', active: false },
+  ]);
+
+  const [appliedData, setAppliedData] = React.useState([
+    { name: 'Option 7', active: false },
+  ]);
+
+  const transfer = (
+    fromData: Array<TransferItemDataType>,
+    setFromData: (data: Array<TransferItemDataType>) => void,
+    setToData: (data: Array<TransferItemDataType>) => void,
+    sendAll?: boolean,
+  ) => {
+    setToData((oldToData) => {
+      const newToData = [...oldToData];
+      const newFromData: Array<TransferItemDataType> = [];
+      fromData.forEach((item) => {
+        if (sendAll || item.active === true) {
+          const newItem = item;
+          newItem.active = false;
+          newToData.push(newItem);
+        } else {
+          newFromData.push(item);
+        }
+      });
+      setFromData(newFromData);
+      return newToData;
+    });
+  };
+
+  return (
+    <TransferList {...args}>
+      <TransferList.ListboxWrapper>
+        <TransferList.ListboxLabel>Options</TransferList.ListboxLabel>
+        <TransferList.Listbox>
+          {optionData?.map((item, index) => {
+            return (
+              <TransferList.Item
+                actionable
+                active={item.active}
+                onActiveChange={(isActive: boolean) => {
+                  setOptionData((oldData) => {
+                    const newData = [...oldData];
+                    const newObject = { ...newData[index] };
+                    newObject.active = isActive;
+                    newData[index] = newObject;
+                    return newData;
+                  });
+                }}
+              >
+                {item.name}
+              </TransferList.Item>
+            );
+          })}
+        </TransferList.Listbox>
+      </TransferList.ListboxWrapper>
+      <TransferList.Toolbar>
+        <IconButton
+          styleType={'borderless'}
```

Comment:
nit: It would be nice if you added label to tool buttons. It would set an example for users.

Reply:
Labels have been added for each of the buttons